### PR TITLE
pid1: cgroup realization fix

### PR DESCRIPTION
**Pull request by poettering**
A fix for #14475 and a replacement for #14554. I think at least, pleas give this a whirl!

### CODE REVIEW

File: src/core/cgroup.c [hunk 1]
```diff
@@ -2335,14 +2335,23 @@ static void unit_add_siblings_to_cgroup_realize_queue(Unit *u) {
         Unit *slice;
 
         /* This adds the siblings of the specified unit and the siblings of all parent units to the cgroup
-         * queue. (But neither the specified unit itself nor the parents.) */
+         * queue. (But neither the specified unit itself nor the parents.)
+         *
+         * Propagation of realization "side-ways" (i.e. towards siblings) is in relevant on cgroup-v1 where
```
Comment (keszybz) [inline src/core/cgroup.c:2340]: "is in relevant"

**Comment by keszybz** [comment]
This fixes #14475.

**Comment by keszybz** [comment]
CentOS CI (Arch in KVM) failed in TEST-43-PRIVATEUSER-UNPRIV, looks unrelated.

File: src/core/cgroup.c (hunk 1 above)
Comment (poettering) [reply src/core/cgroup.c:2340]: https://github.com/systemd/systemd/pull/14571

**Pull request merged**